README.md

```diff
@@ -2,19 +2,28 @@
 
 ⚠️ **Warning: this project is still very alpha**
 
-This project is:
+React component variations are:
 * A format called "variations" for specifying React component **_examples_** and providing **_documentation_** for them.
 * Tools for finding, validating, and iterating over variations.
+* A collection of "consumers" for doing useful things with variations.
 
-A variation looks like:
+## Table of contents
```

Comment:
it might be nice to use a tool like doctoc ([example](https://github.com/nvm-sh/nvm/blob/master/package.json#L17)) plus a CI check, to ensure that this can easily stay up to date.